### PR TITLE
chore(tooling): ignore package.json changes in diff

### PR DESCRIPTION
**Pull request by adamweeks**

### CODE REVIEW

File: tooling/commands/ci.js [hunk 1]
```diff
@@ -45,8 +45,8 @@ module.exports = {
   },
   handler: wrapHandler(async (argv) => {
     let packages = argv.integration ?
-      await updated({dependents: true, npm: true}) :
-      await updated({npm: true});
+      await updated({dependents: true, npm: !!process.env.CI}) :
```
Comment (adamweeks) [inline tooling/commands/ci.js]: This is supposed to run on ci, so can we leave this as a manual `true`?

File: tooling/modified-packages.js [hunk 2]
```diff
@@ -60,8 +60,8 @@ const updated = async ({dependents, npm}) => {
 
 const modified = async (argv) => {
   let changedPackages = argv.integration ?
-    await updated({dependents: true, npm: true}) :
-    await updated({npm: true});
+    await updated({dependents: true, npm: !!process.env.CI}) :
```
Comment (adamweeks) [inline tooling/modified-packages.js]: This should be the only place that checks the `CI` env var

File: tooling/commands/ci.js (hunk 1 above)
Comment (taymoork2) [reply tooling/commands/ci.js]: This file actually doesn't run anymore since we moved to a bash based loop

File: tooling/lib/updated.js [hunk 3]
```diff
@@ -18,7 +18,7 @@ const {diff} = require('./git');
  * npm instead of upstream/master
  * @returns {Promise<Array<string>>}
  */
-exports.updated = async function updated({dependents, npm}) {
+exports.updated = async function updated({dependents, npm = !!process.env.CI}) {
```
Comment (taymoork2) [inline tooling/lib/updated.js]: I'm ok with making this empty instead of defaulted false if there is no value
I just added it cuz it seemed "appropriate"

**Pull request merged**